Modules/posixmodule.c

```diff
@@ -16823,6 +16823,70 @@ os__create_environ_impl(PyObject *module)
 }
 
 
+#ifdef MS_WINDOWS
+/*[clinic input]
+os._get_user_default_environ
+
+Get user default environment string.
+[clinic start generated code]*/
+
+static PyObject *
+os__get_user_default_environ_impl(PyObject *module)
+/*[clinic end generated code: output=6cce8c186a556ef0 input=e15b3d87fce12734]*/
+{
+    HINSTANCE hUserEnv = LoadLibraryW(L"USERENV");
```

Comment:
These functions have been in the API for decades. I'd prefer it if the code included "userenv.h" and either used an inline `#pragma comment(lib, "userenv")` directive, or added "userenv.lib" to the list of "AdditionalDependencies" in "PCbuild/pythoncore.vcxproj" and "PCbuild/_freeze_module.vcxproj".